HealthcareSystem/PatientWebApp/wwwroot/js/malicious_patients.js

```diff
@@ -18,13 +18,41 @@
 });
 
 function addPatient(patient) {
-
+    
+    let btn_type = '';
+    if (patient.isBlocked == false) {
+        btn_type = '<button name="button_block" class="btn btn-danger" style="margin-bottom:10px;" id="' + patient.jmbg
+            + '" onclick="blockPatient(this.id)">Block</button>';
+    }
+    
     let new_patient = $('<div style="margin-top: 10px; margin-left: 21%; margin-bottom:20px; border-style: solid; border-color: black; border-width: 1px; background-color: #cce6ff; padding-top: 40px;left: 450px; top: 200px; width:600px;">'
         + '<table style="height: 10px; margin-left: 10px; margin-bottom: 20px; width: 350px;">'
         + '<tr><td width="25%"><th>Name:</th></td><td>' + ' ' + patient.name + '</td>'
         + '<td width="25%"><th>Surname:</th></td><td>' + ' ' + patient.surname + '</td></tr>'
         + '<tr><td width="25%"><th>Email:</th></td><td>' + ' ' + patient.email + '</td>'
-        + '<td width="25%"><th>Phone:</th></td><td width="30px">' + ' ' + patient.phone + '</td></tr></br>' + ' </table ></div > ');
+        + '<td width="25%"><th>Phone:</th></td><td width="30px">' + ' ' + patient.phone + '</td></tr></br>' 
+
+        + '<tr style="margin-top:20px;"><th></th><td><span style="width: 200px;"></span>' + btn_type + '</td></tr></br></table></div>');
 
     $('div#div_patients').append(new_patient);
-}
\ No newline at end of file
+}
+
+
+function blockPatient(patientJmbg) {
+
+    $.ajax({
+        type: "PUT",
+        url: "/api/patient/blocked/" + patientJmbg,
+        success: function () {
+            alert("You have successfully blocked the patient.");
```

Comment:
Here are some reasons why you should not use alerts in your code:

As you said, a lot alert boxes may annoy the users
Styling of alert boxes is as far as I know impossible. Furthermore every browser displays them diffrent, which may not look good in all cases with your website-design
Most modern / common browsers block alerts after a site tried to display a few of them, to prevent alert spamming
You can use jQuery UI dialogs as an alternative, or anything else that is able to display modal dialogs. That way you can customize / style the dialog box and browsers don't block them.

I generally use alert boxes only for debugging / error reporting, not more, but always remember alerts stop the execution of the code for a purpose. Otherwise, you should be using console.log because alert can actually change the result of your code if your code involves asynchronous logic, so you should make sure that it won't affect any asynchronous logic that you may have running.